<ui version="4.0" >
 <class>CalibrationWindowClass</class>
 <widget class="QWidget" name="CalibrationWindowClass" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Karte kalibrieren</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <item>
    <widget class="CalibrationRenderArea" native="1" name="calibRenderArea" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>320</width>
       <height>320</height>
      </size>
     </property>
     <property name="sizeIncrement" >
      <size>
       <width>32</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="sizeConstraint" >
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="topMargin" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Datei mit Kartengrafik</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mapFileLineEdit" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mapFileOpenPushButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Öffnen ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="CalibrationPointView" name="lonLatTableView" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeIncrement" >
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <property name="sizeConstraint" >
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="calibRestartPushButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Neu beginnen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calibCancelPushButton" >
         <property name="text" >
          <string>Zurücksetzen</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="mapBoundaries" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title" >
        <string>Grenzen</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>links</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>rechts</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>oben</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>unten</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="leftLineEdit" >
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="rightLineEdit" >
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="topLineEdit" >
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLabel" name="bottomLineEdit" >
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calibOkPushButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Speichern und schließen</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CalibrationRenderArea</class>
   <extends>QWidget</extends>
   <header>CalibrationRenderArea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalibrationPointView</class>
   <extends>QTableView</extends>
   <header>CalibrationPointView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
